### PR TITLE
Responsive styles

### PR DESCRIPTION
**Pull request by ivanzusko**

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -9,6 +9,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
     <link rel="stylesheet" href="/static/styles.css">
+    <link rel="stylesheet" href="/static/responsive.css">
```
Comment (tyv) [inline _layouts/default.html]: better keep all in one file or add media queries to link

File: static/styles.css [hunk 2]
```diff
@@ -66,58 +70,58 @@ body {
...
 .header__illustration {
   position: absolute;
   left: 50%;
-  bottom: -20px;
+  bottom: -2rem;
   z-index: -1;
   width: 120%;
   transform: translateX(-50%);
-  filter: grayscale(0.5) saturate(2) blur(6px) opacity(0.8);
+  filter: grayscale(.5) saturate(2) blur(.6rem) opacity(.8);
```
Comment (tyv) [inline static/styles.css]: you don't need to blur image depends on media query (also we didn't use that class name yet)

**Review by tyv — CHANGES_REQUESTED** [review]
it is ok mostly!
you proposed good styles for header -- feel free to add them.

also please give some space for body sides, see screenshot

**Comment by tyv** [comment]
need to add some space:

File: index.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ ref: index
 lang: ua
 prevTitle: Минулі зустрічі
 nextTitle: Майбутні зустрічі
-callForSpeakers: Маєте чим поділитися зі спільнотою? <a href="#write-anchor">Запропонуйте доклад</a>.
+callForSpeakers: Маєте чим поділитися зі спільнотою? <a href="#write-anchor">Запропонуйте доклад</a>
```
Comment (tyv) [inline index.md]: "доповідь"

**Review by tyv — CHANGES_REQUESTED** [review]

File: static/styles.css (hunk 2 above)
Comment (ivanzusko) [reply static/styles.css]: I've just commented this rule out, if it's ok

File: _layouts/default.html (hunk 1 above)
Comment (ivanzusko) [reply _layouts/default.html]: done

**Review by tyv — APPROVED** [review]
🎉

**Pull request merged**